### PR TITLE
Let biome check more files

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: examples/hyperbolic/tree.json [hunk 1]
```diff
@@ -1 +1,699 @@
...
\ No newline at end of file
+[
```
Comment (hhugo) [inline examples/hyperbolic/tree.json:1]: This is not supposed to be pretty printed. It was generated by deriving_json somehow and should stay compact.

File: examples/hyperbolic/image_info.json [hunk 2]
```diff
@@ -1 +1,1628 @@
...ammal","en","Aquatic_mammal"],[0,"Waitoreke","en",""],[0,"Artificial induction of immunity","en","Artificial_induction_of_immunity"],[0,"Electroreception","en",""],[0,"Ornithorhynchidae","en",""],[0,"Platypus","en",""],[0,"Mammifère aquatique","fr","Mammif%C3%A8re_aquatique"],[0,"Faune de l'Australie","fr","Faune_de_l%27Australie"],[0,"Murray (fleuve)","fr","Murray_(fleuve)"],[0,"Monotremata","fr",""],[0,"Ornithorynque","fr",""],[0,"Mammifère","fr","Mammif%C3%A8re"]],[0,"http://upload.wikimedia.org/wikipedia/commons/f/f2/Platypus.jpg"]],[0,"echidne","Short-beaked_Echidna_Tasmania.jpg","I, <a target=\"_blank\" href=\"http://commons.wikimedia.org/wiki/User:KeresH\">KeresH</a> [<a target=\"_blank\" href =\"http://www.gnu.org/copyleft/fdl.html\">GFDL</a> or <a target=\"_blank\" href =\"http://www.creativecommons.org/licenses/by-sa/3.0/\">CC-BY-SA-3.0</a> or <a target=\"_blank\" href =\"http://www.creativecommons.org/licenses/by-sa/2.5-2.0-1.0\">CC-BY-SA-2.5-2.0-1.0</a>]",800,523,[0,[0,"Échidné à nez court","fr","%C3%89chidn%C3%A9_%C3%A0_nez_court"]],[0,"http://upload.wikimedia.org/wikipedia/commons/thumb/8/80/Short-beaked_Echidna_Tasmania.jpg/800px-Short-beaked_Echidna_Tasmania.jpg"]]]
```
Comment (hhugo) [inline examples/hyperbolic/image_info.json:1]: This was generated automatically and should stay compact.

**Comment by hhugo** [comment]
I don't think we should format json file yet.

**Comment by smorimoto** [comment]
Indeed, I just fixed that

**Pull request merged**